snap/bin/stop-edgex.sh

```diff
@@ -95,12 +95,30 @@ if [ $SUPPORT_LOGGING = "y" ] ; then
     fi
 fi
 
+if [ $SECURITY ]; then
+    # stop kong - note this might fail, so we just force it to always succeed with "|| true"
+    $SNAP/bin/kong-wrapper.sh stop -p $SNAP_DATA/kong || true
+
+    # killing nginx is tricky, as we need to make sure we don't inadvertantly kill any other nginx
+    # processes, so we use egrep with a more specific pattern
+    nginxppid=`ps -ef | egrep "nginx.*${SNAP_DATA}/kong" | grep -v grep | awk '{print $2}'`
```

Comment:
You should use **pgrep** for this, and the other similar commands below...

Reply:
I agree, pgrep is a better way to do this, but for the time being can we hold off on these and other shell cleanups for another shellcheck PR where we do this kind of thing everywhere?